build/dependencies.props

```diff
@@ -1,14 +1,14 @@
 <Project>
   <PropertyGroup>
-    <CoreFxVersion>4.4.0-*</CoreFxVersion>
+    <CoreFxVersion>4.4.0</CoreFxVersion>
```

Comment:
The lineup PR is already queued in my local changes. This one bled over from it as I'm doing the work as incrementally as possible. I can remove it for the sake of purity.